src/main.zig

```diff
@@ -8,11 +8,10 @@ fn draw(renderer: sdl.Renderer, fbuf: [32]u64) anyerror!void {
     while (k < 64) : (k += 1) {
         var j: u7 = 0;
         while (j < fbuf.len) : (j += 1) {
-            // std.log.info("{}, {}, {}, {}", .{ j, k, fbuf[j], (fbuf[j] & (@intCast(u32, 1) << @truncate(u5, k))) });
-            if ((fbuf[j] & (@intCast(u64, 1) << @truncate(u6, 64 - k))) != 0) {
+            if ((fbuf[j] & (@as(u64, 1) << @truncate(u6, 64 - k))) != 0) {
                 try renderer.fillRect(sdl.Rectangle{
-                    .x = @intCast(c_int, k) * 10,
-                    .y = @intCast(c_int, j) * 10,
+                    .x = @as(c_int, k) * 10,
+                    .y = @as(c_int, j) * 10,
```

Comment:
Note that you're casting a `u7` to a `c_int`. You could very well omit any coercion here, being that it is highly unlikely that an emulator is ever going to be run on a platform where c_int is any smaller than 8 bits.

Reply:
Technically speaking, since `c_int` is a platform-dependent type, the "correct" thing to do would be to use `@intCast`, but this is such an edge case that it's really up to your discretion.